aoc_day_2/src/part2.rs

```diff
@@ -42,7 +42,6 @@ where P: AsRef<Path>, {
 
```

Comment:
I remember github reviews letting you comment on untouched code, but this apparently has changed over the years.

Congradulations on your progress!

A note on the above:
```rust
    let input_filename = &args[1]; 
 ```
 1 may not exist. it would be nice to panic with an error message if that were so, like 
```rust
args.nth(1).unwrap_or_else(panic!("provide a filename"));
```